### PR TITLE
chore: [sc-28630] Test every CLARK route to make sure it is reachable from existing Gateway

### PR DESCRIPTION
**Pull request by Cwagne17**
Story details: https://app.shortcut.com/clarkcan/story/28630

### CODE REVIEW

File: .terraform-docs.yml [hunk 1]
```diff
@@ -9,35 +9,32 @@ content: |-
...
   ```hcl
-  module "example" {
-    {{"\n"}}
-    {{"\t"}} source  = "github.com/Cyber4All/terraform-cyber4all-catalog//modules/<REPLACE_WITH_MODULE>?ref=v<REPLACE_WITH_VERSION>"
-    {{"\n"}}
+  terraform {
+  {{"\t"}} source = "github.com/Cyber4All/terraform-cyber4all-catalog//modules/<REPLACE_WITH_MODULE>?ref=v<REPLACE_WITH_VERSION>"
+  }
 
+  inputs = {
```
Comment (mfranz1) [inline .terraform-docs.yml:16]: Lol this was all we needed to fix the inputs error?

File: modules/ecs-service/main.tf [hunk 2]
```diff
@@ -201,7 +201,7 @@ locals {
   }
 
   portMappings = !var.create_scheduled_task && var.enable_service_connect ? [{
-    name          = random_id.service_connect[0].hex
+    name          = sha256(var.ecs_service_name)
```
Comment (mfranz1) [inline modules/ecs-service/main.tf:204]: You're encrypting the ecs service name?

File: modules/ecs-service/variables.tf [hunk 3]
```diff
@@ -101,6 +101,12 @@ variable "ecs_task_cpu" {
   default     = 256
 }
 
+variable "ecs_task_ephemeral_storage" {
+  type        = number
+  description = "The amount of ephemeral storage (in GiB) to allocate to the ECS task."
+  default     = 21
```
Comment (mfranz1) [inline modules/ecs-service/variables.tf:107]: Wait, how? 21 GiB? Shouldn't we give it more?

File: .terraform-docs.yml (hunk 1 above)
Comment (Cwagne17) [reply .terraform-docs.yml:16]: Yep its below

File: modules/ecs-service/variables.tf (hunk 3 above)
Comment (Cwagne17) [reply modules/ecs-service/variables.tf:107]: That's the default, we'd override for clark-bundling-service

File: modules/ecs-service/main.tf (hunk 2 above)
Comment (Cwagne17) [reply modules/ecs-service/main.tf:204]: Hashing just to make a unique id that isn't tied to another resource lifecycle

**Pull request merged**